hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/libhdfspp/include/hdfspp/block_location.h

```diff
@@ -19,6 +19,8 @@
 #ifndef HDFSPP_BLOCK_LOCATION_H
 #define HDFSPP_BLOCK_LOCATION_H
 
+#include <stdint.h>
```

Comment:
there is a dedicated PR that just opened to fix it, https://github.com/apache/hadoop/pull/7644. I would suggest doing it separately to make the backport easier.

Reply:
Thanks for pointing this. Have removed related changes from this PR.